pandas/core/indexes/base.py

```diff
@@ -63,6 +63,8 @@ def _try_get_item(x):
 
 
 def _make_comparison_op(op, cls):
+
+    # @ops.unpack_and_defer
```

Comment:
`Index.__eq__(Series)` doesn't return NotImplemented, kind of inconvenient special case